### PR TITLE
Test examples as part of CI run

### PR DESCRIPTION
**Pull request by sectioneight**
Technically this should probably be `/...` but seems like overkill for now.

### CODE REVIEW

File: examples/simple/main.go [hunk 1]
```diff
@@ -29,9 +29,8 @@ import (
 
 func main() {
 	svc, err := service.WithModules(
-		uhttp.New(registerHTTPers, []uhttp.Filter{simpleFilter{}}),
+		uhttp.New(registerHTTPers),
```
Comment (yutongp) [inline examples/simple/main.go]: is this change intended?

Comment (sectioneight) [inline examples/simple/main.go]: Sorta, I meant to include it to get the build passing but now I merged your real fix and rebased.

File: Makefile [hunk 2]
```diff
@@ -142,6 +142,7 @@ examples:
 	@$(call label,Building examples)
 	@echo
 	$(ECHO_V)$(MAKE) -C examples/keyvalue ECHO_V=$(ECHO_V)
+	$(ECHO_V)go test ./examples/simple
```
Comment (madhuravi) [inline Makefile]: why not follow the same format as keyvalue? shouldn't we run the same commands to test both?

**Comment by sectioneight** [comment]
A makefile seems overkill to run a single test command. The make rules in
key value are much more complex. I can move this command to a separate
Makefile, but that feels like overkill to me right now.

**Comment by sectioneight** [comment]
All tests passed after re-triggering in Travis
